Update Netatmo integration documentation (#12447)
home-assistant/home-assistant.github.io: source/_integrations/netatmo.markdown

The `netatmo` sensor platform is consuming the information provided by a [Netatmo Weather Station](https://www.netatmo.com/en-us/weather/weatherstation) or a
[Netatmo Home Coach](https://www.netatmo.com/en-us/aircare/homecoach) [Netatmo](https://www.netatmo.com) device.

## Webhooks

> The smart indoor and outdoor cameras as well as the smart smoke alarm send instant events to Home Assistant by using webhooks. It is required to have your camera enabled in Home Assistant.

Reviewer: ```suggestion
The smart indoor and outdoor cameras, as well as the smart smoke alarm, send instant events to Home Assistant by using webhooks. It is required to have your camera enabled in Home Assistant.
```

> To be able to receive events from Netatmo, your Home Assistant instance needs to be accessible from the web. To acchive this you can either use your Nabu Casa account or ([Home Assistant instructions](/addons/duckdns/)) and you need to have the base_url configured for the HTTP integration ([documentation](/integrations/http/#base_url)).

Reviewer: ```suggestion
To be able to receive events from Netatmo, your Home Assistant instance needs to be accessible from the web. To achieve this you can either use your Nabu Casa account or ([Home Assistant instructions](/addons/duckdns/)) and you need to have the base_url configured for the HTTP integration ([documentation](/integrations/http/#base_url)).
```

> Events coming in from Netatmo will be available as events in Home Assistant and are fired as netatmo_*, along with their data. You can use this event to trigger automations.

Reviewer: ```suggestion
Events coming in from Netatmo will be available as events in Home Assistant and are fired as `netatmo_*`, along with their data. You can use these events to trigger automations.
```
